eng/common/templates/steps/execute-sdl.yml

```
@@ -33,7 +33,7 @@ steps:
 
 - ${{ if ne(parameters.overrideParameters, '') }}:
   - powershell: ${{ parameters.executeAllSdlToolsScript }} ${{ parameters.overrideParameters }}
-    displayName: Execute SDL
+    displayName: Execute SDL (Overridden)
```

Maybe something like this? I think the `(Overriden)` might also mean "this step didn't run" or "the results of this step are not taken into account"

```suggestion
    displayName: Customized SDL
```

---

I am suggesting this change to be align with other similar steps: "Install Guardian" and "Install Guardian (Overridden)". And for some reason two jobs have the same display name "Execute SDL" --> "Execute SDL" & "Execute SDL (Overridden)"

---

Ah, if it's a convention already then yeah